pkg/payload/precondition/clusterversion/upgradeable.go

```diff
@@ -111,18 +113,93 @@ func (pf *Upgradeable) Run(ctx context.Context, releaseContext precondition.Rele
 // Name returns Name for the precondition.
 func (pf *Upgradeable) Name() string { return "ClusterVersionUpgradeable" }
 
+//  RecentEtcdBackup checks if a recent etcd backup has been taken.
+type RecentEtcdBackup struct {
+	key      string
+	lister   configv1listers.ClusterVersionLister
```

Comment:
nit: now that we have two listers, can this one be `cvLister`?

Reply:
done